main.tex

```diff
@@ -293,8 +295,8 @@ \subsection*{\hypertarget{Ability}{Ability and statistic cards}}
     \includegraphics[width=0.48\textwidth]{images/ability.PNG}
 
 \end{wrapfigure}
-Every ability and statistic card has a basic effect and a stronger expert effect, which is shown below the basic effect. Whenever a player plays an ability or statistic card, they must choose which effect they are using. The number of expert \includegraphics[scale=0.5]{images/expert.PNG} effects a player can use each round is limited by their main hero’s \hyperlink{Level}{level}.\\[6pt]
-\textbf{Important}: Certain cards are limited to the Necropolis faction \includegraphics[scale=1]{images/necro.PNG}. When a non-Necropolis player draws one, they may either discard it and draw a new card as a replacement or gain it as normal. Non-Necropolis players cannot use faction specific cards from their hand in any way besides discarding them.
+Every ability and statistic card has a basic effect and a stronger expert effect, which is shown below the basic effect. Whenever a player plays an ability or statistic card, they must choose which effect they are using. The number of expert \includesvg[height=10px]{svgs/expert.svg} effects a player can use each round is limited by their main hero’s \hyperlink{Level}{level}.\\[6pt]
```

Comment:
Can you delete those PNG files? I guess we won't be needing them any more.

Reply:
Sure thing